src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionQRInfoViewController.swift

```diff
@@ -76,4 +83,16 @@ class ExposureSubmissionQRInfoViewController: DynamicTableViewController, Footer
 		dynamicTableViewModel = viewModel.dynamicTableViewModel
 		tableView.separatorStyle = .none
 	}
+	
+	private func setupNavigationBar() {
+		// set the bar tint to white
+		parent?.navigationController?.navigationBar.tintColor = .white
```

Comment:
why do we color the navBar tint to white? Was this not the actual problem that it should be NOT white?
Or did I got the Jira Ticket description wrong?

Reply:
This is on viewWillDisappear and we want the navigation bar tint to white again on this screen.

![Simulator Screen Shot - iPhone 12 - 2021-05-12 at 17 04 50](https://user-images.githubusercontent.com/77438487/117972208-444d5980-b344-11eb-96cf-e747ec052730.png)

Reply:
but white may only work correct in light mode? Did you tested it in dark mode too?

Reply:
Yes.

![Simulator Screen Shot - iPhone 12 - 2021-05-12 at 17 15 47](https://user-images.githubusercontent.com/77438487/117973398-c8eca780-b345-11eb-9c10-daa32b36e00a.png)

Reply:
Ah okay. 
But then the method should be named like "restoreNavigationbar" or something similar.

Or even better: Do this in the Controller, where it is needed. Otherwise you implement logic into another controller for this one.

Reply:
I have renamed the function